.appveyor.yml

```diff
@@ -0,0 +1,44 @@
+platform:
+  - x64
+
+branches:
+  only:
+  - master
```

Comment:
Added to skip needless CI builds when doing PRs against this repo. Can be dropped if this is moved to an org.